README.md

```diff
@@ -76,6 +78,87 @@ fastify.register(require('fastify-rate-limit'), {
 })
 ```
 
+
+### Options on the endpoint itself
+
+You can overide the default value set during the registration of the plugin from the route declaration, the values will be used `only` for this route.
+
+Regarding the `whitelist`. There are 2 types :
+ - global : will affect all the endpoint.
+ - endpoint : will affect only the targeted endpoint.
+
+`The Global whitelist` is set in the plugin when registering it with fastify.register(...).
+
+`The Endpoint whitelist` is set on the endpoint directly with the { config : rateLimit : whitelist : [] }object.
+
+you need to set the option `whiteListInRedis` to `true` when registering the plugin. This is only available when using redis. 
+
+
+The ACL is checking based on the value of the key from the keyGenerator. In this example we are checking the Ip but it could be a whitelist of specific user id.
```

Comment:
```suggestion
ACL checking is performed based on the value of the key from the `keyGenerator`. In this example we are checking the IP address, but it could be a whitelist of specific user identifiers.
```